### DISCUSSION TITLE
How to enable autowrapping?

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [LazyVim/LazyVim#1959] [Q&A] [answered]
How do I enable wrap in LazyVim? Currently if a line is longer than the screen width, it'll keep on going until the end. I want to disable that behavior.
Is there a plugin that can automatically break lines > 80 textwidth?

**Comment by dpetka2001** [comment, marked as answer]
`<leader>uw` is the keymap to toggle wrapping if i remember correctly. If you want to set it permanently, just set it in your `options.lua` file.

**Reply by limaceous-bushwhacker** [reply]
Thank you, putting `opt.wrap = true` in `options.lua` did the job.

**Reply by centuryhopper** [reply]
i tried that and it said 'undefined global opt' 😭

UPDATE:
I got it. It's
```
local opt = vim.opt
opt.wrap = true
```

vim is the globally recognized variable

**Reply by leikoilja** [reply]
you can also do it in one line ofc, placing this into `options.lua`:
`vim.opt.wrap = true`

**Reply by nxtkofi** [reply]
keymap doesn't work anymore

**Reply by rosscondie** [reply]
`vim.opt.wrap = true` worked for me in the `options.lua` file
